src/System.ServiceModel.Primitives/ref/System.ServiceModel.Primitives.cs

```diff
@@ -379,6 +379,14 @@ public partial class InvalidMessageContractException : System.Exception
         public InvalidMessageContractException(string message) { }
         public InvalidMessageContractException(string message, System.Exception innerException) { }
     }
+
+    [System.AttributeUsageAttribute((System.AttributeTargets)(384), AllowMultiple = false, Inherited = false)]
```

Comment:
Although we haven't done it elsewhere, I really don't like using the magic numbers and casting them to the relevant enum type. The original value was ServiceModelAttributeTargets.MessageMember which isn't public, but that equates to AttributeTargets.Property | AttributeTargets.Field.